### ISSUE TITLE
cannot using <ion-social-markdown> icon

### ISSUE THREAD

**Issue by X-G** [suda/toolbar-main#5]
This is a great package!
I want to add a customized bottom to toggle markdown preview. using
```
    @toolBar.addButton
      icon: 'social-markdown'
      callback: 'markdown-preview:toggle'
      tooltip: 'markdown preview'
      iconset: 'ion'
```
But I found this icon cannot displayed properly.

**Comment by jerone** [comment]
> But I found this icon cannot displayed properly.

I can't not confirm. What do you mean with "cannot displayed properly"?
![2015-05-22 17_38_58-toolbar json - c__users_jerone_ atom - atom](https://cloud.githubusercontent.com/assets/55841/7774160/7fbd3ab6-00a9-11e5-865f-3d4ccf40bf7f.png)

**Comment by X-G** [comment]
![a](https://cloud.githubusercontent.com/assets/6829403/7812910/a35495a4-03b6-11e5-889e-c41fc766f58c.png)
It shows like this. I also found not only  `social-markdown` cannot display.
I'm not sure this is the problem of  `toolbar main` or `tool-bar`.

**Comment by jerone** [comment]
Does the Developer Tools show missing resources?

**Comment by X-G** [comment]
No, there isn't.
I found that almost all the icons in the subset `ion-iso-...` and some of in `ion-social-...` don't work.

**Comment by jerone** [comment]
This is probably fixed by updating to Ionicons 2.0.0. https://github.com/suda/tool-bar/pull/52

Please reopen if after next ToolBar update problem still exists.

**Comment by X-G** [comment]
Thanks!
